### ISSUE TITLE
Cannot start: ImportError: No module named exttype

### ISSUE THREAD

**Issue by hyperknot** [ITCase/pyramid_sacrud#106]
Updated to latest pyramdid sacrud (2.0.0) + sacrud (1.3.4). Cannot start now:
```
File ".../sacrud_deform/__init__.py", line 14, in <module>
    from sacrud.exttype import ChoiceType
ImportError: No module named exttype
```

It's happening with sacrud 1.3.4, isn't happening with 1.3.3.

**Comment by prmtl** [comment]
:+1: 
I can confirm that this issue exists. `sacrud.exttype` module was removed in 0.3.3 but `pyramid_sacrud` was not updated.

**Comment by uralbash** [comment]
@hyperknot @prmtl Thanks for report bug! Fixed it in last version of sacrud_deform==0.1.1 and pyramid_sacrud==0.2.0.post0 . I'll doing redesign pyramid_sacrud architecture (like this http://pyramid-sacrud.readthedocs.org/en/latest/pages/contribute/architecture.html). So if you have problems, please notify them to me.

**Comment by uralbash** [comment]
I close issue, hope now it works.
